x/nft/simulation/operations_test.go

```diff
@@ -0,0 +1,115 @@
+package simulation_test
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/bank/testutil"
```

Comment:
This is only in [v0.46.0-alpha1](https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.46.0-alpha1?tab=versions) and above of the cosmos-sdk and we need to use v0.44.3 everywhere else, so this is a bit problematic.

Reply:
should be solved with the protocgen.sh that uses third_party/proto